electron/games.ts

```diff
@@ -217,8 +217,8 @@ class LegendaryGame {
     let prefix = `--wine-prefix '${fixedWinePrefix.replaceAll("'", '')}'`
 
     const isProton =
-      wineVersion.name.startsWith('Proton') ||
-      wineVersion.name.startsWith('Steam')
+      wineVersion.name.indexOf('Proton') >= 0 ||
```

Comment:
But when this changes? Before, all Protons were Proton, Steam was removed. I missed this on your latest PR then. I dont think we should use Steam and Lutris, only Wine and Proton, like before.

Reply:
Oh? I didn't know. Yeah you missed it probably. I split it up to "Steam Proton", "Lutris Wine" etc.

Reply:
It was like this at first and then I changed it to be more simple to understand. :)
Because then, now, you need to do 2 checks for isProton instead of one.
For the user also is more information.

Reply:
I think indicating where the Wine version comes from may be a good idea. Not sure though, I'm okay with changing it back.

Reply:
Please change it back. It is also easier for us to maintain. I think Proton and Wine are the best indications we will have.

Reply:
Done.